examples/tests/sdk_tests/util/sleep.w

```diff
@@ -10,5 +10,6 @@ test "sleep 100 mili seconds" {
   util.sleep(oneHundredMiliseconds);
   let end = JSHelper.getTime();
   let delta = end - start;
+  log("slept for ${delta} ms");
   assert(delta > 80 && delta < 220);
```

Comment:
imo, the test should only verify that the sleep happened. It's too hard to guarantee the precision.

```suggestion
   assert(delta >= 100);
```